src/main/resources/hibernate-cfg.xml

```diff
@@ -0,0 +1,13 @@
+<hibernate-configuration>
+  <session-factory>
+    <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+    <property name="connection.url">jdbc:mysql://localhost:3306/phones?serverTimezone=UTC</property>
+    <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+    <property name="connection.username">root</property>
+    <property name="connection.password">java123</property>
+    <property name="show_sql ">true</property>
```

Comment:
```suggestion
    <property name="show_sql">true</property>
```
in each hw you have the same whitespace here
and each time you need to remove it after review"

let's be more attentive)